src/pages/home/Form.jsx

```diff
@@ -19,6 +20,13 @@ const SignUpSchema = Yup.object().shape({
 
 const Signup = () => {
 	const [hasSubmitted, hasSubmittedHandler] = useState(false);
+	const submittedMessages = {
```

Comment:
This should be declared outside of render.

Reply:
Ok, I moved it outside `Signup()` (I think that's what this means?)